internal/sirius/payment.go

```diff
@@ -0,0 +1,36 @@
+package sirius
+
+import "fmt"
+
+type Payment struct {
+	ID          int           `json:"id,omitempty"`
+	CaseID      int           `json:"case_id,omitempty"`
+	Source      PaymentSource `json:"source"`
+	Amount      FeeString     `json:"amount"`
+	PaymentDate DateString    `json:"paymentdate"`
+	Type        TypeOfPayment `json:"type"`
+	CreatedDate DateString    `json:"createddate"`
+	Locked      bool          `json:"locked,omitempty"`
+	CreatedByID int           `json:"createdby_id"`
+}
```

Comment:
I don't think these are quite right: they should all be camelcase (e.g. `paymentDate`) and there are some I wouldn't expect the API to return (e.g. CaseID)

Also it's better not to include fields you don't use (e.g. CreatedByID) because it gives the impression they're necessary and makes them harder to remove from the API in the future.

Reply:
Yeah you're right, i'll just include those fields that I need right now and update to camel case :)